tms-gRPC-Service/interfaces/iConfig.ts

```diff
@@ -0,0 +1,26 @@
+export interface IConfig {
+  functionName: string;
+  port: number;
+  redisDB: string;
+  redisAuth: string;
+  redisHost: string;
+  redisPort: number;
+  redisConnection: boolean;
+  apmServiceName: string;
+  apmSecretToken: string;
+  apmURL: string;
+  dbName: string;
+  dbURL: string;
+  dbUser: string;
+  dbPassword: string;
+  graphName: string;
+  collectionName: string;
+  dev: string;
+  transactionRoutingHostname: string;
```

Comment:
TMS doesn't need to know about transaction routing... Also not Redis, Arango. please cleanup.

Reply:
Updated ✅